doc/adv-manual/special-features.rst

```diff
@@ -142,28 +142,23 @@ PointSources/AreaSources/MultiPointSources, fault sources are
 untouched.
 
 
-Ruptures in TOML format
+Ruptures in CSV format
 -------------------------------------------
 
-Since engine v3.8 there is a way to serialize ruptures in
-TOML_ format. The command to give is::
+Since engine v3.10 there is a way to serialize ruptures in
+CSV format. The command to give is::
   
-  $ oq extract rupture/<rup_id> <calc_id>`
+  $ oq extract ruptures?min_mag=<mag> <calc_id>`
```

Comment:
On zsh (default shell on macOS), the query string needs to be quoted:
`$ oq extract "ruptures?min_mag=<mag>" <calc_id>`

Reply:
Fixed the docs